src/web/components/Figure.tsx

```diff
@@ -150,11 +144,23 @@ const decidePosition = (role: RoleType | 'richLink') => {
 	}
 };
 
+type Props = {
+	children: React.ReactNode;
+	isMainMedia: boolean;
+	role?: RoleType | 'richLink';
```

Comment:
maybe worth adding 'richLink' to `RoleType`

Reply:
Yeah it's a weird one. `richLink` isn't a genuine CAPI role, but seems to be used here as if it is one (albeit with a quite specific name!). I didn't attempt to change this approach in the PR but agree it is potentially confusing.

Reply:
just noticed `'richLink'` was here before ^^ still prob worth adding it maybe? (though could be spun out into a new PR)